packages/p2p/src/components/buy-sell/buy-sell-form.jsx

```diff
@@ -179,134 +222,154 @@ const BuySellForm = ({ advert, handleClose, handleConfirm, setErrorMessage, setI
                 setSubmitForm(submitForm);
 
                 return (
-                    <Form noValidate>
-                        <div className='buy-sell__popup-content'>
-                            <div className='buy-sell__popup-field-wrapper'>
-                                <div className='buy-sell__popup-field'>
-                                    <span className='buy-sell__popup-info--title'>
-                                        {is_buy_advert ? localize('Seller') : localize('Buyer')}
-                                    </span>
-                                    <div className='buy-sell__popup-info--text'>{advertiser_name}</div>
-                                </div>
-                                <div className='buy-sell__popup-field'>
-                                    <span className='buy-sell__popup-info--title'>
-                                        {localize('Rate (1 {{ currency }})', { currency: account_currency })}
-                                    </span>
-                                    <div className='buy-sell__popup-info--text'>
-                                        {getFormattedText(price, local_currency)}
+                    <React.Fragment>
+                        <Form noValidate>
+                            <div className='buy-sell__modal-content'>
+                                <div className='buy-sell__modal-field-wrapper'>
+                                    <div className='buy-sell__modal-field'>
+                                        <Text as='p' color='less-prominent' line_height='m' size='xxs'>
```

Comment:
You're using the Text component with the same properties in a couple of places, would it be better to create another component that returns this with a `message` or `text` prop?

Reply:
imo it will just add another level of abstraction which is unnecessary.